packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -188,7 +188,7 @@ describe('HCP home page', () => {
 
     const select = within(addPatientDialog).getByTestId('patient-team-selector')
     fireEvent.mouseDown(within(select).getByRole('button'))
-    fireEvent.click(screen.getByRole('option', { name: teamTwo.name }))
+    fireEvent.click(screen.getByRole('option', { name: teamThree.name }))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why this change?

Reply:
Why not ?